### PR TITLE
6.2.1

### PR DESCRIPTION
**Pull request by calvinmetcalf**
what I like to see, only tests added

### CODE REVIEW

**Comment by mcollina** [comment]
Tests failing :( some of them because of Error: ENOENT: no such file or directory, open '/home/travis/build/nodejs/readable-stream/test/fixtures/keys/ec-key.pem'

**Comment by calvinmetcalf** [comment]
we don't actually need that file, it's for the c api

**Comment by lpinca** [comment]
browser failures are probably caused by ngrok limits on free plan. Also there is bug in the ngrok module that prevents open tunnels from closed, see https://github.com/defunctzombie/zuul/issues/251#issuecomment-223586880.

In light of #217 it might makes sense to switch back to localtunnel (it worked fine for me in the last few days.)

**Comment by calvinmetcalf** [comment]
ok trying without ngrok

**Comment by mcollina** [comment]
@calvinmetcalf why don't we ask ngrok if they could sponsor an account for CI to the Foundation? Or maybe we can buy one. 

Another solution is to setup our own localtunnel server, I think we should try to avoid that as far as possible (but it's a solution).

**Comment by lpinca** [comment]
Another option is to use karma + karma-sauce-launcher instead of zuul but the ppc issue will not be solved. I don't think there is a ppc binary for sauce-connect.

**Comment by lpinca** [comment]
I think zuul can also run with sauce-connect.

**Comment by calvinmetcalf** [comment]
@lpinca thanks will try that

**Comment by calvinmetcalf** [comment]
woot that works thanks @lpinca that being said there is a suble timing bug in one of the tests in 0.8 that just for fun, can be prevented by putting certain console.log statments in place

**Comment by calvinmetcalf** [comment]
ok from what I can tell, in node 0.8 setTimeout(fn, 0) will sometimes comeback before process.nextTick

**Comment by calvinmetcalf** [comment]
more specifically I think setTimeout(fn, 0) is the same as process.nextTick, making it setTimeout(fn, 1) fixes things

**Comment by calvinmetcalf** [comment]
well 1 doesn't but 4 does

**Comment by mcollina** [comment]
I think we should drop 0.8 asap and bump the major version.

**Comment by calvinmetcalf** [comment]
naw I got 0.8 working

File: package.json [hunk 1]
```diff
@@ -17,13 +17,12 @@
...
     "tape": "~4.5.1",
-    "zuul": "~3.10.0",
-    "zuul-ngrok": "^4.0.0"
+    "zuul": "~3.10.0"
   },
   "scripts": {
     "test": "tap test/parallel/*.js test/ours/*.js",
     "browser": "npm run write-zuul && zuul -- test/browser.js",
-    "write-zuul": "printf \"ui: tape\ntunnel: ngrok\nbrowsers:\n  - name: $BROWSER_NAME\n    version: $BROWSER_VERSION\n\">.zuul.yml",
+    "write-zuul": "printf \"ui: tape\nsauce_connec: true\nbrowsers:\n  - name: $BROWSER_NAME\n    version: $BROWSER_VERSION\n\">.zuul.yml",
```
Comment (lpinca) [inline package.json]: There is a typo here: `sauce_connec` → `sauce_connect`.

Comment (calvinmetcalf) [inline package.json]: fixed thanks

**Comment by sonewman** [comment]
hmm, there does seem to still be an issue :(

**Comment by lpinca** [comment]
@sonewman if you are talking about CI, yes zuul is flaky as hell and `sauce_connect` make it even more flaky :/

As @mcollina said, it would be great if we can get a ngrok account.

**Comment by calvinmetcalf** [comment]
**sign** yeah I think your right

**Comment by mcollina** [comment]
We should probably update this to 6.2.2 while we are at it https://github.com/nodejs/readable-stream/issues/220.

**Comment by calvinmetcalf** [comment]
ok added #220

**Comment by calvinmetcalf** [comment]
though in this case it was because I a new es6 feature was introduced

**Comment by calvinmetcalf** [comment]
bumped it, i don't think there is anything actually wrong with the pull, just the tests

**Comment by mcollina** [comment]
There is this change: https://github.com/nodejs/readable-stream/pull/218/files#diff-ba6a0df0f5212f5cba5ca5179e209a17L544.

**Comment by calvinmetcalf** [comment]
yeah I take this back a browser is failing legitimately

**Comment by calvinmetcalf** [comment]
testing manually to see if I can reproduce the android error

**Comment by calvinmetcalf** [comment]
weird not able to reproduce the android error when running saucelabs locally

**Comment by lpinca** [comment]
@calvinmetcalf I didn't have issues with localtunnel lately, maybe try to switch again? 😄

**Comment by calvinmetcalf** [comment]
ok all browsers have had at least one green run so I'm calling this green enough

**Pull request merged**